src/server/graphql/models/Team/teamMutation.js

```diff
@@ -18,14 +19,15 @@ export default {
     },
     async resolve(source, {newTeam}, {authToken}) {
       // require cachedUserId in the input so an admin can also create a team
-      const userId = newTeam.leader.cachedUserId;
+      const userId = newTeam.leader[0].cachedUserId;
```

Comment:
@mattkrick: I noticed you spec'd the leader as a list. Any reason you can think to have more than one leader?

If not, I'd like to play by highlander rules for now, "there can be only one."